spec/models/order_spec.rb

```diff
@@ -2,7 +2,16 @@
 
 RSpec.describe Order, type: :model do
   let(:user) { Fabricate(:user) }
-  let(:order) { Order.create(user_id: user.id, status: 0) }
+  let(:item) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.